olive/systems/olive_system.py

```diff
@@ -34,7 +34,7 @@ def run_pass(
         raise NotImplementedError()
 
     @abstractmethod
-    def evaluate_model(self, model: OliveModel, metrics: List[Metric]) -> Dict[str, Any]:
+    def evaluate_model(self, model: OliveModel, metrics: List[Metric], device: Device = Device.CPU) -> Dict[str, Any]:
```

Comment:
The `device` is already part of the system (although aml and docker don't have it right now). But they will be added in Mike's PR. 

Can we remove the `device` from the `evaluate_model` signature? We can refer to `self.device` in the LocalSystem when calling `evaluator.evaluate`

Reply:
Done!